python/ray/rllib/agents/ddpg/ddpg_policy_graph.py

```diff
@@ -499,59 +388,185 @@ def set_weights(self, weights):
 
     @override(PolicyGraph)
     def get_state(self):
-        return [TFPolicyGraph.get_state(self), self.cur_epsilon]
+        return [
+            TFPolicyGraph.get_state(self), self.cur_noise_scale,
+            self.cur_pure_exploration_phase
+        ]
 
     @override(PolicyGraph)
     def set_state(self, state):
         TFPolicyGraph.set_state(self, state[0])
         self.set_epsilon(state[1])
```

Comment:
As we have changed the var name from 'cur_epsilon' to 'cur_noise_scale', it may be better to change 'set_epsilon' to 'set_noise_scale' accordingly.

Reply:
I kept `set_epsilon()` because that method gets and called from `DQNTrainer`, which is the parent of `DDPGTrainer`. I agree the naming inconsistency is ugly, though; would it be acceptable to rename `set_epsilon` to `set_exploration_value` in both the DQN case _and_ the DDPG case, then add a comment to both methods explaining what "exploration value" means in both cases? (for DQN it would be epsilon, for DDPG it would be noise scaling factor)